rust/common-runtime/src/bin/runtime.rs

```diff
@@ -1 +1,30 @@
-fn main() {}
+#[macro_use]
+extern crate tracing;
+
+use std::net::SocketAddr;
+
+use common_runtime::{serve, CommonRuntimeError};
+use tracing_subscriber::{fmt::Layer, layer::SubscriberExt, EnvFilter, FmtSubscriber};
+
+#[tokio::main]
+pub async fn main() -> Result<(), CommonRuntimeError> {
+    let subscriber = FmtSubscriber::builder()
+        .with_env_filter(EnvFilter::from_default_env())
+        .finish();
+    tracing::subscriber::set_global_default(subscriber.with(Layer::default().pretty()))
```

Comment:
I've noticed `Layer::default().pretty()` results in duplicate logs -- consider `tracing::subscriber::set_global_default(subscriber)`